### DISCUSSION TITLE
An improvement regarding the version of libraries in LRZ

### DISCUSSION THREAD

**Discussion by XinxuZhao** [tum-esm/stilt-at-esm#11] [Improvements]
https://github.com/tum-esm/stilt-at-esm/blob/4e861b2e8b2120b8118c0fa6e97b33fdd19a04e1/install.sh#L13

Hi Moritz,
I open this discussion regarding the version of modules in HPC of LRZ. LRZ regularly updated its system once per year. Now the default r version is above 4. Maybe for this commend, we can change to 'module load r', which means that every time the running will search for the current default version.

By the way, I am considering reusing our own nodes in CoolMUC3, instead of running STILT in CoolMUC2. I will let you know whether there is some difference.

Cheers

Xinxu

**Comment by dostuffthatmatters** [comment]
I didn't know that we had our own nodes :D Is there documentation about them?
